### PR TITLE
data_base

### PR DESCRIPTION
**Pull request by MarianDiakiv**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -2,3 +2,7 @@ spring.datasource.url=${DATASOURCE_URL}
 spring.datasource.username=${DATASOURCE_USER}
 spring.datasource.password=${DATASOURCE_PASSWORD}
 spring.datasource.driver-class-name=org.postgresql.Driver
+
+spring.jpa.hibernate.ddl-auto=update
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQL9Dialect
+spring.jpa.show-sql=true
```
Comment (lhalam) [inline src/main/resources/application.properties:8]: add endline

File: src/main/java/com/softserve/teachua/entity/City.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+@Table(name = "city")
+public class City {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String city;
+
+    @OneToMany(mappedBy = "city")
+    private List<ChildrenCenter> childrenCenters;
+
```
Comment (lhalam) [inline src/main/java/com/softserve/teachua/entity/City.java:23]: remove empty lines

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -2,3 +2,7 @@ spring.datasource.url=${DATASOURCE_URL}
 spring.datasource.username=${DATASOURCE_USER}
 spring.datasource.password=${DATASOURCE_PASSWORD}
 spring.datasource.driver-class-name=org.postgresql.Driver
```
Comment (lhalam) [inline src/main/resources/application.properties:4]: add all parameters to system variables

File: src/main/java/com/softserve/teachua/entity/enams/ChildrenAge.java [hunk 4]
```diff
@@ -0,0 +1,5 @@
+package com.softserve.teachua.entity.enams;
+
+public enum ChildrenAge {
+ZERO_FOUR,FOUR_SIX,SIX_ELEVEN,TWELVE_SIXTEEN
```
Comment (lhalam) [inline src/main/java/com/softserve/teachua/entity/enams/ChildrenAge.java:4]: format this code

File: src/main/java/com/softserve/teachua/entity/City.java (hunk 2 above)
Comment (MarianDiakiv) [reply src/main/java/com/softserve/teachua/entity/City.java:22]: done

File: src/main/resources/application.properties (hunk 3 above)
Comment (MarianDiakiv) [reply src/main/resources/application.properties]: done

File: src/main/java/com/softserve/teachua/entity/enams/ChildrenAge.java (hunk 4 above)
Comment (MarianDiakiv) [reply src/main/java/com/softserve/teachua/entity/enams/ChildrenAge.java]: done

File: src/main/resources/application.properties (hunk 1 above)
Comment (MarianDiakiv) [reply src/main/resources/application.properties]: done

File: src/main/java/com/softserve/teachua/entity/Activities.java [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+@Entity
+@Table
+public class Activities {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String activity;
+    @ManyToMany(mappedBy = "activities")
+    private Set<Club> club;
+}
```
Comment (lhalam) [inline src/main/java/com/softserve/teachua/entity/Activities.java:22]: add endline

File: src/main/java/com/softserve/teachua/entity/ChildrenCenter.java [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String addres;
+    private String coordinates;
+    @ManyToOne
+    @JoinColumn(name = "city_id")
+    private City city;
+    @OneToMany(mappedBy = "childrenCenter")
+    private List<Club> clubs;
+}
```
Comment (lhalam) [inline src/main/java/com/softserve/teachua/entity/ChildrenCenter.java:26]: add endline

File: src/main/java/com/softserve/teachua/entity/City.java [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+@Entity
+@Table(name = "city")
+public class City {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String city;
+    @OneToMany(mappedBy = "city")
+    private List<ChildrenCenter> childrenCenters;
+}
```
Comment (lhalam) [inline src/main/java/com/softserve/teachua/entity/City.java:22]: add endline

**Review by lhalam — DISMISSED** [review]

**Review by softservedata — APPROVED** [review]

**Review by lhalam — APPROVED** [review]

File: src/main/java/com/softserve/teachua/entity/ChildrenCenter.java [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+@Getter
+@Setter
+@NoArgsConstructor
+@Entity
+@Table(name = "children_center")
+public class ChildrenCenter {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String addres;
```
Comment (ikohutua) [inline src/main/java/com/softserve/teachua/entity/ChildrenCenter.java:19]: should be 'address'

**Review by terrytarik — APPROVED** [review]

**Pull request merged**